public/app/features/teams/TeamGroupSync.test.tsx

```diff
@@ -1,56 +1,72 @@
 import { render, screen } from 'test/test-utils';
 
-import { setBackendSrv } from '@grafana/runtime';
-import { setupMockServer } from '@grafana/test-utils/server';
-import { MOCK_TEAMS } from '@grafana/test-utils/unstable';
-import { backendSrv } from 'app/core/services/backend_srv';
-import { TeamGroup, TeamState } from 'app/types/teams';
+import { TeamGroup } from 'app/types/teams';
 
 import TeamGroupSync from './TeamGroupSync';
+import * as hooks from './hooks';
 import { getMockTeamGroups } from './mocks/teamMocks';
 
-setBackendSrv(backendSrv);
-setupMockServer();
```

Comment:
We should leave the mock server and backend setup in here, and instead add the mock endpoints to the `@grafana/test-utils` package